tests/e2e/features/slideViewer.feature

```diff
@@ -0,0 +1,13 @@
+Feature: markdown presentation viewer
+  As a user
+  I want to view markdown documents in presentation format
+  So that I can easily present content in a structured manner
+  
+  Background:
+    Given user "Admin" has logged in
+
+
+  Scenario: preview markdown file in presentation viewer
+    Given user has uploaded the markdown file "test-markdown.md"
```

Comment:
when not mentioning the user name it should be `the user`
see also https://doc.owncloud.com/server/next/developer_manual/testing/acceptance-tests.html#specifying-the-actor